src/content/docs/iast/introduction.mdx

```diff
@@ -0,0 +1,130 @@
+
+New Relic IAST helps address some of the limitations of more traditional application security tools like Static Application Security Testing (SAST), Dynamic Application Security Testing (DAST), and penetration testing that struggle to keep pace with modern application delivery, especially applications based on based on distributed microservices. IAST helps bridge the gap, saving you money and time, as it's substantially less likely to generate false positive findings, which increase developer confidence and drive efficiency. You get double duty from your manual or automated QA tests to continuously detect and prioritize vulnerabilities that are exploitable, along with guidance on how to eliminate the risk.
+
+IAST helps you:
+
+- Find and fix exploitable vulnerabilities that matter in your code efficiently without waiting for scan results.
+- Ship code faster with unmatched detection accuracy of security risks and substantially fewer false positives.
+- See and secure every application you build and run with full visibility into your code, web components, and configuration data.
+- Reduce the time and cost to eliminate vulnerabilities.
+- Cut down the noise from both static application security testing (SAST) and dynamic application security testing (DAST) analysis.
+
+IAST is fully integrated with [New Relic Vulnerability Management](/docs/vulnerability-management/overview/), allowing you to continuously and quickly find, fix, and verify high-risk vulnerabilities across the software development lifecycle. IAST's exclusive proof of exploits helps developers use their time more efficiently by eliminating the time wasted chasing false positives, reproducing vulnerabilities, and working low-impact security risks.
+
+New Relic IAST is delivered by our Application Performance Management (APM) agents and is enabled by a simple configuration setting change. IAST is available for the following languages:
+
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                Language
+            </th>
+            <th>
+                APM agent version
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                [Java](https://github.com/newrelic/csec-java-agent#support-matrix)
+            </td>
+            <td>
+                8.4.0 or higher
+            </td>
+        </tr>
+        <tr>
+            <td>
+                [NodeJS](https://github.com/newrelic/csec-node-agent/#supported--modules)
+            </td>
+            <td>
+                10.3.0 or higher
+            </td>
+        </tr>
+        <tr>
+            <td>
+                [Go](https://github.com/newrelic/csec-go-agent/tree/v0.1.0#support-matrix)
+            </td>
+            <td>
+                3.23.0 or higher
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+
+Follow [these instructions](/docs/iast/install/) to install New Relic IAST.
+
+## About Vulnerability Severity [#vuln-severity]
```

Comment:
Yes! And can this have a more descriptive title? Why does someone need to know about vulnerability severity? Is it to help them read their charts/dashboards/results?